squad-get-log-lines

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import os
+import sys
+from pathlib import Path
+
+from squad_client.core.api import SquadApi
+from squad_client.core.models import Squad, TestRun
+from squad_client.utils import getid
+
+from squadutilslib import get_file
+
+squad_host_url = "https://qa-reports.linaro.org/"
+SquadApi.configure(cache=3600, url=os.getenv("SQUAD_HOST", squad_host_url))
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Script to scrape the log parameter from tests in specified projects."
+    )
+
+    parser.add_argument(
+        "--group",
+        required=True,
+        help="The name of the SQUAD group.",
+    )
+
+    parser.add_argument(
+        "--projects",
+        required=True,
+        action="extend",
+        nargs="+",
+        help="A list of SQUAD projects to fetch logs from.",
+    )
+
+    parser.add_argument(
+        "--suites",
+        required=True,
+        action="extend",
+        nargs="+",
+        help="A list of SQUAD test suite names to get logs for.",
+    )
+
+    parser.add_argument(
+        "--debug",
+        required=False,
+        action="store_true",
+        default=False,
+        help="Display debug messages.",
+    )
+
+    parser.add_argument(
+        "--full-log",
+        required=False,
+        action="store_true",
+        default=False,
+        help="Download the full log in addition to the log lines.",
+    )
```

Comment:
Can this be default True ?
We don't really need to pull out any boot/test-log-parser tests lines.
Since that can be done if we just pull out the logfiles and run the logparser locally once.
Then change the regexes and rerun it locally, now you have two different runs of the logparser run that can be compared and see if regex changes was an improvement or not before committing it.

The script will be way simpler if we drop the boot/test-log-parser tests and only pull the complete logfiles.